python 3.8 windows build error with related to use of `_Pragma`

from win64 python 3 conda-forge auto build

```
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3180): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3180): error C2440: 'initializing': cannot convert from 'const char [20]' to 'int'
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3180): note: There is no context in which this conversion is possible
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3180): error C2146: syntax error: missing ';' before identifier '_Pragma'
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3182): error C2143: syntax error: missing ';' before '{'
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3182): error C2447: '{': missing function header (old-style formal list?)
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3235): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3235): error C2374: '_Pragma': redefinition; multiple initialization
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3180): note: see declaration of '_Pragma'
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3238): error C2448: '_Pragma': function-style initializer appears to be a function definition
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3241): error C2065: 'PyConduit_DataType_TYPE': undeclared identifier
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3249): error C2065: 'PyConduit_DataType_TYPE': undeclared identifier
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3455): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3455): error C2374: '_Pragma': redefinition; multiple initialization
%SRC_DIR%\src\libs\conduit\python\conduit_python.cpp(3180): note: see declaration of '_Pragma'
```

It should be supported in visual studio -- not sure what is wrong:

https://docs.microsoft.com/en-us/cpp/preprocessor/pragma-directives-and-the-pragma-keyword?view=msvc-160

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,14 @@ Notable changes to Conduit are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project aspires to adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+#### General
+- Avoid compile issue with using `_Pragma()` with Python 3.8 on Windows
+
+
 ## [0.7.2] - Released 2021-05-19
 
 ### Added

--- a/src/libs/conduit/python/conduit_python.cpp
+++ b/src/libs/conduit/python/conduit_python.cpp
@@ -33,7 +33,9 @@
 // we suppress the deprecated warning only in 3.8.
 // 
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_PUSH_DEP_DECL \
      _Pragma("GCC diagnostic push") \
      _Pragma("GCC diagnostic ignored \"-Wdeprecated-declarations\"")
@@ -41,7 +43,9 @@
 #define PRAGMA_PUSH_DEP_DECL
 #endif
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_POP_DEP_DECL _Pragma("GCC diagnostic pop")
 #else
 #define PRAGMA_POP_DEP_DECL

--- a/src/libs/relay/python/conduit_relay_io_python.cpp
+++ b/src/libs/relay/python/conduit_relay_io_python.cpp
@@ -54,7 +54,9 @@ using namespace conduit::relay::io;
 // we suppress the deprecated warning only in 3.8.
 // 
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_PUSH_DEP_DECL \
      _Pragma("GCC diagnostic push") \
      _Pragma("GCC diagnostic ignored \"-Wdeprecated-declarations\"")
@@ -62,7 +64,9 @@ using namespace conduit::relay::io;
 #define PRAGMA_PUSH_DEP_DECL
 #endif
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_POP_DEP_DECL _Pragma("GCC diagnostic pop")
 #else
 #define PRAGMA_POP_DEP_DECL

--- a/src/libs/relay/python/conduit_relay_mpi_python.cpp
+++ b/src/libs/relay/python/conduit_relay_mpi_python.cpp
@@ -54,7 +54,9 @@ using namespace conduit::relay::mpi;
 // we suppress the deprecated warning only in 3.8.
 // 
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_PUSH_DEP_DECL \
      _Pragma("GCC diagnostic push") \
      _Pragma("GCC diagnostic ignored \"-Wdeprecated-declarations\"")
@@ -62,7 +64,9 @@ using namespace conduit::relay::mpi;
 #define PRAGMA_PUSH_DEP_DECL
 #endif
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_POP_DEP_DECL _Pragma("GCC diagnostic pop")
 #else
 #define PRAGMA_POP_DEP_DECL

--- a/src/libs/relay/python/conduit_relay_web_python.cpp
+++ b/src/libs/relay/python/conduit_relay_web_python.cpp
@@ -54,7 +54,9 @@ using namespace conduit::relay::web;
 // we suppress the deprecated warning only in 3.8.
 // 
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_PUSH_DEP_DECL \
      _Pragma("GCC diagnostic push") \
      _Pragma("GCC diagnostic ignored \"-Wdeprecated-declarations\"")
@@ -62,7 +64,9 @@ using namespace conduit::relay::web;
 #define PRAGMA_PUSH_DEP_DECL
 #endif
 
-#if PY_VERSION_HEX >= 0x03080000 && PY_VERSION_HEX < 0x03090000
+#if PY_VERSION_HEX >= 0x03080000 && \
+    PY_VERSION_HEX < 0x03090000 && \
+    !defined(CONDUIT_PLATFORM_WINDOWS)
 #define PRAGMA_POP_DEP_DECL _Pragma("GCC diagnostic pop")
 #else
 #define PRAGMA_POP_DEP_DECL

